diff -up lxqt-config-1.4.0/CMakeLists.txt.1~ lxqt-config-1.4.0/CMakeLists.txt
--- lxqt-config-1.4.0/CMakeLists.txt.1~	2023-11-05 18:19:59.949130516 +0100
+++ lxqt-config-1.4.0/CMakeLists.txt	2023-11-05 18:22:15.365217289 +0100
@@ -8,6 +8,8 @@ project(lxqt-config)
 
 set(LXQT_CONFIG_PROJECT "${PROJECT_NAME}")
 
+set(CMAKE_CXX_STANDARD 20)
+
 option(UPDATE_TRANSLATIONS "Update source translation translations/*.ts files" OFF)
 option(WITH_APPEARANCE "Build the 'lxqt-config-appearance'" ON)
 option(WITH_BRIGHTNESS "Build the 'lxqt-config-brightness'" ON)
diff -up lxqt-config-1.4.0/lxqt-config-monitor/CMakeLists.txt.1~ lxqt-config-1.4.0/lxqt-config-monitor/CMakeLists.txt
--- lxqt-config-1.4.0/lxqt-config-monitor/CMakeLists.txt.1~	2023-11-05 12:36:31.000000000 +0100
+++ lxqt-config-1.4.0/lxqt-config-monitor/CMakeLists.txt	2023-11-05 18:19:59.949130516 +0100
@@ -3,6 +3,8 @@ project(lxqt-config-monitor)
 find_package(KF5Screen ${KF5SCREEN_MINIMUM_VERSION} REQUIRED)
 find_package(Qt5Svg ${QT_MINIMUM_VERSION} REQUIRED)
 
+set(CMAKE_CXX_STANDARD 20)
+
 set(HEADERS
     monitorsettingsdialog.h
     monitor.h
